### PR TITLE
feat: 新增'观点网'

### PR DESCRIPTION
**Pull request by drgnchan**
## 该 PR 相关 Issue / Involved issue

Close #

## 完整路由地址 / Example for the proposed route(s)

```routes
/guandian/finance
/guandian/business
```

## 新 RSS 检查列表 / New RSS Script Checklist
  
- [x] 新的路由 New Route
  - [x] 跟随 [v2 路由规范](https://docs.rsshub.app/joinus/script-standard.html) Follows [v2 Script Standard](https://docs.rsshub.app/en/joinus/script-standard.html)
- [x] 文档说明 Documentation
  - [x] 中文文档 CN
  - [ ] 英文文档 EN
- [ ] 全文获取 fulltext
  - [x] 使用缓存 Use Cache
- [ ] 反爬/频率限制 anti-bot or rate limit?
  - [ ] 如果有, 是否有对应的措施? If yes, do your code reflect this sign?
- [ ] [日期和时间](https://docs.rsshub.app/joinus/pub-date.html) [date and time](https://docs.rsshub.app/en/joinus/pub-date.html)
  - [x] 可以解析 Parsed
  - [x] 时区调整 Correct TimeZone
- [ ] 添加了新的包 New package added
- [ ] `Puppeteer`

### CODE REVIEW

**Comment by drgnchan** [comment]
>

File: lib/v2/guandian/radar.js [hunk 1]
```diff
@@ -0,0 +1,11 @@
+module.exports = {
+    'guandian.cn': {
+        _name: '观点网',
+        www: {
+            title: '观点网 - 资讯',
```
Comment (TonyRL) [inline lib/v2/guandian/radar.js:5]:
```suggestion
            title: '资讯',
```

File: lib/v2/guandian/index.js [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+            const link = a.attr('href');
+            const title = a.find('h3').text();
+            const desc = item.find('.con_l_con_r p').text();
+
+            const dateStr = item.find('#keyword span').text();
+            return {
+                title,
+                link,
+                description: desc,
+                pubDate: parseDate(dateStr),
```
Comment (TonyRL) [inline lib/v2/guandian/index.js:27]:
```suggestion
                pubDate: timezone(parseDate(dateStr), 8),
```
```js
const timezone = require('@/utils/timezone');
```

**Comment by drgnchan** [comment]
@TonyRL Thankyou.
Polish done.

**Pull request merged**